converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java

```diff
@@ -3,6 +3,8 @@
 
 import java.io.UncheckedIOException;
 import java.nio.charset.Charset;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.nio.charset.Charset'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)